cumulus/client/relay-chain-rpc-interface/src/metrics.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Cumulus.
+
+// Cumulus is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Cumulus is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
+
+use prometheus::{Error as PrometheusError, HistogramTimer, Registry};
+use prometheus_endpoint::{HistogramOpts, HistogramVec, Opts};
+
+// Gathers metrics about the blockchain RPC client.
+#[derive(Clone)]
+pub(crate) struct BlockchainRpcMetrics {
```

Comment:
maybe `RelaychainRpcMetrics` would be better?

Reply:
here: [4481f5e](https://github.com/paritytech/polkadot-sdk/pull/5572/commits/4481f5ebf73172752a568ed1afe81776ded9be13)